features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Shared/AppsAndKeys/Tokens.jsx

```diff
@@ -53,229 +53,179 @@ const styles = theme => ({
+                        value={scopesSelected}
+                        onChange={e => this.handleOnChange(e)}
+                        input={<Input id='select-multiple-chip' />}
+                        renderValue={selected => (
+                            <div className={classes.chips}>
+                                {selected.map(value => (
                                     <Chip key={value} label={value} className={classes.chip} />
-                                    ))}
-                                </div>
-                                )}
-                                MenuProps={MenuProps}
-                            >
-                                {allScopes.map(scope => (
-                                <MenuItem
-                                    key={scope}
-                                    value={scope}
-                                >
-                                    {scope}
-                                </MenuItem>
                                 ))}
-                            </Select>
-                            <Typography variant="caption">When you generate access tokens to APIs protected by scope/s, you can select the scope/s and then generate the token for it. Scopes enable fine-grained access control to API resources based on user roles. You define scopes to an API's resources. When a user invokes the API, his/her OAuth 2 bearer token cannot grant access to any API resource beyond its associated scopes.</Typography>
-                        </FormControl>
+                            </div>
+                        )}
+                        MenuProps={MenuProps}
+                    >
+                        {subscriptionScopes.map(scope => (
+                            <MenuItem
+                                key={scope}
+                                value={scope}
+                            >
+                                {scope}
+                            </MenuItem>
+                        ))}
+                    </Select>
+                    <Typography variant='caption'>
+                        <FormattedMessage
+                            id='subscription.scopes.description'
+                            defaultMessage={'When you generate access tokens to APIs protected by scope/s,' +
+                            ' you can select the scope/s and then generate the token for it. Scopes enable ' +
+                            'fine-grained access control to API resources based on user roles. You define scopes to ' +
+                            'an API resource. When a user invokes the API, his/her OAuth 2 bearer token cannot grant ' +
+                            'access to any API resource beyond its associated scopes.'}
+                        />
+
+
+                    </Typography>
+                </FormControl>
             </React.Fragment>
         );
     }
 }
-
+Tokens.contextTypes = {
+    intl: PropTypes.object.isRequired,
```

Comment:
Please change this as

```
intl: PropTypes.shape({}).isRequired,
```

`PropTypes.object` is not allowed in default eslint configs

Reply:
Fixed